docs/cookbook/html-conversion.md

```diff
@@ -638,30 +644,30 @@ export default () => {
   // check that the admin-bar module exists
   const loggedIn = !!window.apos.modules['@apostrophecms/admin-bar'];
   if (loggedIn) {
-    // wrap in a time out to give the admin bar object time to load the bar and return the height
-    setTimeout(() => {
+    // wrap in `apos.util.onReady()` that fires when the page is loaded and at every refresh
+    apos.util.onReady(() => {
       //get the admin-bar height
       const adminBarHeight =
         window.apos.modules['@apostrophecms/admin-bar'].height;
         // get the navigation ID - if you are using a different template, adjust accordingly
       const pageNav = document.getElementById('mainNav');
-      // set the position of the navigation to after the admin-bar
+      // set the absolute position of the navigation to after the admin-bar
       pageNav.style.top = adminBarHeight + 'px';
-    }, 10);
+    });
   }
 };
 
 ```
 
 <template v-slot:caption>
-modules/@apostrophecms/asset/ui/apos/apps/adminBarHeight.js
+modules/asset/ui/apos/apps/adminBarHeight.js
```

Comment:
I would capitalize "admin" here for consistency with the way we name our "apps" in the admin UI.

Reply:
Done.